cfpb/cfpb-chart-builder
# Prep library for Wagtail's chart block

Changes needed for https://github.com/cfpb/cfgov-refresh/pull/2688.

## Additions

- Added a `color` field but it's not in use yet.

## Changes

- Updated docs in readme.
- Changed `group` attribute to `metadata`.
- Bumped version to 1.0.0.

## Testing

- Pull down this branch.
- `npm install`
- `npm start`
- Open http://localhost:8088/test and confirm that nothing is broken.
